packages/react-components/src/index.ts

```diff
@@ -20,6 +20,9 @@ export {
 /* @conditional-compile-remove(rooms) */
 export type { _Permissions, _PermissionsProviderProps, Role } from './permissions';
 
+/* @conditional-compile-remove(pinned-participants) */
```

Comment:
no need to remove it here.
This export statement should be removed from `communication-react` package index

Reply:
done